package.json

```diff
@@ -3,26 +3,21 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "firstbuild": "npm install && fable",
-    "prebuild": "npm install",
-    "build": "fable --watch"
+    "build": "rollup -c -w"
   },
   "dependencies": {
     "buffer": "^5.0.0",
     "core-js": "^2.4.1",
-    "fable-core": "^0.7.23",
-    "fable-elmish": "^0.8.0",
-    "fable-elmish-react": "^0.8.1",
-    "fable-powerpack": "^0.0.19",
-    "fable-react": "^0.8.0",
     "react": "~15.3.2",
     "react-native": "^0.36.0",
     "react-native-image-picker": "^0.25.1",
     "react-native-onesignal": "^1.2.3",
     "react-native-windows": "^0.36.0"
   },
   "devDependencies": {
-    "fable-compiler": "^0.7.31",
-    "react-native-cli": "^2.0.1"
```

Comment:
I deleted the local dependency because the [React Native instructions](https://facebook.github.io/react-native/docs/getting-started.html) (see Building Projects with Native Code) recommend the global installation, an also [the build script](https://github.com/fable-compiler/fable-react_native-demo/blob/744cd3432abe349f01ec903c0cb39418e59f58e4/build.fsx#L49) of this repo calls the global `react-native-cli` (at least for macOS/Linux), so that's why I deleted the local dependency to avoid confusion.

However, I guess that installing it locally works too (I haven't checked it myself). We could put the dependency back and just fix the build script.